sdk/python/kfp/v2/components/types/type_utils.py

```diff
@@ -33,6 +33,9 @@
     'markdown': artifact_types.Markdown,
 }
 
+_GOOGLE_TYPES_PATTERN = r'^google.[A-Za-z]+$'
+_GOOGLE_TYPES_VERSION = '0.0.1'
```

Comment:
not blocking - what would be the long term plan for specifying the artifact types in YAML?

Reply:
I'm thinking about a dictionary like type: `{google.VertexModel: {version: 0.0.1}}`